README.md

```diff
@@ -3,7 +3,12 @@ The beginnings of a Sculpin-based style guide creation tool.
 
 ## Dependencies
 1. NodeJS 
+   1. To intall NodeJS using Homebrew run `brew install node` 
 1. [Sculpin](https://sculpin.io/getstarted/)
+    1. `curl -O https://download.sculpin.io/sculpin.phar`
+    2. `chmod +x sculpin.phar`
+    3. `mv sculpin.phar sculpin` or if you are using vagrant run `sudo -H mv sculpin.phar /bin/sculpin` and skip step 4
+    4. `mv sculpin ~/bin/` 
```

Comment:
@becw would you recommend getting rid of the instructions here and adding the note that it depends on a projects sculpin or is that already part of your PR? 


I see conflicts on the horizob